components/common/usage_percent_bar/index.tsx

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Curious about having this in a folder, but only the index file.

Reply:
0/5 on this. Components often grow to have subcomponents so if that happens its a little less file shuffling to add the components. If it doesn't happen, then we have slightly more nested components than we need.